e2etest/bigint_test.go

```diff
@@ -141,7 +143,7 @@ func prepare(bc blockchain.Blockchain, sf factory.Factory, ap actpool.ActPool, e
 	r.NoError(err)
 	selp, err := action.Sign(elp, priKey)
 	r.NoError(err)
-	r.NoError(ap.Add(context.Background(), selp))
+	r.Error(ap.Add(context.Background(), selp))
```

Comment:
why it has error now?

Reply:
because the amount cannot be negative now